README.md

```diff
@@ -187,18 +182,77 @@ This will be resolved from the first of:
 
 <br/>
 
+
+## FAQ
+
+#### I've run `git machete discover`... but the branch layout I see in `.git/machete` doesn't exactly match what I expected. Am I doing something wrong?
+
+[//]: # (For how to find Medium header anchors, see https://www.freecodecamp.org/news/how-to-link-to-a-specific-paragraph-in-your-medium-article-2018-table-of-contents-method-e66595fea549/)
+No! It's all right, `discover` is based on an (imperfect)
+[heuristic](https://medium.com/virtuslab/git-machete-strikes-again-traverse-the-git-rebase-jungle-even-faster-with-v2-0-f43ebaf8abb0#0544)
```

Comment:
Although, to be fair, Medium allows also linking to individual paragraphs (`<p>`) via these ids, like here... so maybe not so bad this time!

Reply:
Yeah, that's why stable section identifiers generate in a predictable way out of section title (as in most markdown renderers) is what the _civilized world_ does... but not Medium ofc